csv_export_payment/models/csv_export_payment.py

```diff
@@ -27,12 +27,15 @@ class PartnerCSVExport(models.TransientModel):
     def get_domain(self):
         # est-ce que les paiments peuvent rester en draft plusieurs jours?
         #  => non, les paiements ne passent pas par l'état brouillon.
-        return [
-            ("journal_id.type", "=", "cash"),
-            ("state", "!=", "draft"),
-            ("create_date", ">=", self.start_date),
-            ("create_date", "<", self.end_date),
-        ]
+        domain = [("state", "!=", "draft"), ("state", "!=", "cancel")]
```

Comment:
Is it a request to remove the condition `("journal_id.type", "=", "cash")`?

Reply:
No it's a mistake !

Reply:
fixed